### PR TITLE
Add events API v2

### PR DESCRIPTION
**Pull request by se-bastiaan**
Part of #1428

### Summary
Add events API v2

### How to test
Steps to test the changes you made:
1. Check the API docs
2. Check the endpoints for the events

### CODE REVIEW

File: website/events/api/v2/views.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from events.api.v2.serializers.event import EventSerializer
+from events.api.v2.serializers.event_registration import EventRegistrationSerializer
+from events.models import Event, EventRegistration
+
+
+class EventListView(ListAPIView):
+    """Returns an overview of all upcoming events."""
+
+    serializer_class = EventSerializer
+    queryset = Event.objects.all()
```
Comment (se-bastiaan) [inline website/events/api/v2/views.py:14]: Need to add filters and searching

**Comment by KiOui** [comment]
deploy review

File: website/thaliawebsite/api/v2/serializers/html.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class CleanedHTMLSerializer(serializers.BaseSerializer):
+    def to_internal_value(self, data):
+        pass
+
+    def update(self, instance, validated_data):
+        pass
+
+    def create(self, validated_data):
+        pass
```
Comment (KiOui) [inline website/thaliawebsite/api/v2/serializers/html.py:15]: Shouldn't these call the super methods?

**Comment by KiOui** [comment]
remove review

File: website/thaliawebsite/api/v2/serializers/html.py (hunk 2 above)
Comment (se-bastiaan) [reply website/thaliawebsite/api/v2/serializers/html.py]: Fixed

**Comment by se-bastiaan** [comment]
deploy review

**Review by KiOui — APPROVED** [review]

**Pull request merged**